libraries/eosiolib/contracts/eosio/action.hpp

```diff
@@ -447,8 +447,20 @@ namespace eosio {
     * trans_action.send(st.issuer, to, quantity, memo);
     * @endcode
     */
-   template <eosio::name::raw Name, auto Action>
+   template <eosio::name::raw Name, auto Action, eosio::name::raw DefaultContract = ""_n>
    struct action_wrapper {
+      constexpr action_wrapper()
+         : code_name(DefaultContract), permissions() {}
+
+      constexpr action_wrapper( eosio::name user )
+         : code_name(DefaultContract), permissions({1,{user,"active"_n}}) {}
```

Comment:
If we're going to use this shortcut of using a name instead of a `permission_level` and assuming `active`, then it should probably be generalized.